### DISCUSSION TITLE
"DevRel Practitioner" Definition - What outlines the criteria?

### DISCUSSION THREAD

**Discussion by anajsana** [DevRel-Foundation/governance#63] [Ideas]
As this initiative evolves (and given the fact that DevRel Foundation operates as a non-funded, community-driven model) I think getting community consensus + having a document on what outlines the criteria and responsibilities for joining the DevRel Foundation as a practitioner member / community member is important for future elections that might arise in this community.

FinOps has a nice example. [here](https://github.com/finopsfoundation/foundation/blob/main/practitioner_definition.md#finops-practitioner-definition). Maybe a similar document can be drafted and discussed in this thread and iterated in upcoming community calls, to include as part of the governance repo. Thoughts?

**Comment by fer-correa** [comment]
I believe the idea is very good, I will try with the regional vision of LATAM to make a contribution of thought: this would be of great value, also thinking about guiding those who still have doubts about what DevRel is. Believe it or not, this is still one of the biggest pains in part of LATAM (what it is, what it does, why it exists...).
So the definition of practitioner can also make clear “what is really expected of a person to consider them a practitioner”.

**Comment by goofmint** [comment]
Depending on the criteria, I feel that this could exclude those who are more deeply committed to DevRel as a business. So the criteria is very important.

via FinOps.
```
(ii) not be employed by a FinOps, cloud, or cloud management vendor;
```

**Reply by fer-correa** [reply]
Yeah... I agree

**Reply by anajsana** [reply]
That's a good point @goofmint !

I think having documentation around criteria doesn’t need to be used only as a way to exclude segments of people for this context. IMHO, by making sure there's documented (and explicit) criteria, It could also actively encourage certain groups to feel included — like those in DevRel business, community manager roles, etc — 

PS: This is also somehow connected with one of the last comments during the [last APAC & EMEA call](https://github.com/DevRel-Foundation/governance/discussions/62#discussioncomment-10559088) that Samson brought up:  _some people might be doing DevRel without using the term. When presenting, consider alternative words like “community management.”_

**Reply by jerdog** [reply]
To that end, it is why at the DevRel Collective we defined it the way we did

https://devrelcollective.fun/#devrel-definition

**Reply by anajsana** [reply]
Thanks for sharing this resource @jerdog ! Is there a repo under creative commons license like cc-by 4.0  that can use to reference? There is an idea to send a pull request for a `glossary.md` with the aim to have that space for the community to contribute with terminology. I think that referencing the work that devrel collective has done for devrel definition + any other resource / definition that already exists would be great.

**Comment by hhkaos** [comment]
I think this is an excellent idea, and at some point, it will have to be exclusive, which I believe is necessary. Where to draw that line? That's a more complex discussion 😅 .

That being said, as Jorge Barrachina highlights in [his book](https://www.amazon.com/DevRel-Puzzle-Jorge-Barrachina-Guti%C3%A9rrez/dp/B0CHL9MZQS), I understand DevRel as a discipline more than a role, encompassing many roles such as developer advocate, developer evangelist, developer relations strategist, technical community manager, etc.
